src/reporter/errors/templates.js

```diff
@@ -261,6 +261,30 @@ export default {
         ${getScreenshotInfoStr(err)}
     `,
 
+    [TYPE.incorrectGlobalWaitForActionEventArgument]: err => dedent`
+        ${getMsgPrefix(err, CATEGORY.actionError)}Error at step <span data-type="step-name">${err.stepName}</span>:
+
+        <code data-type="api">__waitFor</code> action's first parameter should be a function.
+
+        ${getScreenshotInfoStr(err)}
+    `,
+
+    [TYPE.incorrectGlobalWaitForActionTimeoutArgument]: err => dedent`
+        ${getMsgPrefix(err, CATEGORY.actionError)}Error at step <span data-type="step-name">${err.stepName}</span>:
+
+        <code data-type="api">__waitFor</code> action's "timeout" parameter should be a positive number.
+
+        ${getScreenshotInfoStr(err)}
+    `,
+
+    [TYPE.globalWaitForActionTimeoutExceeded]: err => dedent`
+        ${getMsgPrefix(err, CATEGORY.timeout)}Error at step <span data-type="step-name">${err.stepName}</span>:
+
+        <code data-type="api">__waitFor</code> user code timeout exceeded.
```

Comment:
And we have this approach (without `:`) in all other error descriptions: https://github.com/georgiy-abbasov/testcafe-phoenix/blob/implement-global-wait-error-types/src/reporter/errors/templates.js